src/Login/Login.js

```diff
@@ -0,0 +1,57 @@
+import React, { Component } from 'react';
+import ApiCalls from '../ApiCalls.js';
+import App from '../App/App.js';
+import './Login.css';
+
+
+class Login extends Component {
+  constructor() {
+    super();
+    this.state = {
+      email: '',
+      password: ''
+      //name: 'Olivia', email: 'olivia@turing.io', password: 'pword'
+    }
+  }
+
+  getUserInput = event => {
+    this.setState({ [event.target.name]: event.target.value })
+  }
+
+  submitLogin = event => {
+    event.preventDefault();
+    const loginInfo = {
```

Comment:
You can actually just put state in the handleSubmit! `this.props.handleSubmit(this.state)`